### PR TITLE
topsql: add aggregate by table or db

### PR DESCRIPTION
**Pull request by lilin90**
### What is changed, added or deleted? (Required)

Adds aggregate by table or db 

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):** 

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/18664 
- Other reference link(s): https://github.com/pingcap/tidb/issues/55540

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dashboard/top-sql.md [hunk 1]
```diff
@@ -25,11 +25,7 @@ Top SQL is suitable for analyzing performance issues. The following are some typ
 
 Top SQL cannot be used to pinpoint non-performance issues, such as incorrect data or abnormal crashes.
```
Comment (hfxsd) [inline dashboard/top-sql.md:26]:
```suggestion
```

**Review by hfxsd — APPROVED** [review]

File: dashboard/top-sql.md [hunk 2]
```diff
@@ -71,31 +68,39 @@ The following are the common steps to use Top SQL.
...
-Additionally, you can configure Top SQL as follows:
+    You can adjust the time range in the time picker or select a time range in the chart to get a more precise and detailed look at the problem. A smaller time range can provide more detailed data, with precision of up to 1 second.
+
+    ![Change time range](/media/dashboard/top-sql-usage-change-timerange.png)
+
+    If the chart is out of date, you can click the **Refresh** button or select Auto Refresh options from the **Refresh** drop-down list.
+
+    ![Refresh](/media/dashboard/top-sql-usage-refresh.png)
+
+6. View the CPU resource usage of TiKV by table or database level to quickly identify resource usage at a higher level.
```
Comment (lilin90) [inline dashboard/top-sql.md:95]:
```suggestion
6. View the CPU resource usage by table or database level to quickly identify resource usage at a higher level. Currently, only TiKV instances are supported.
```

File: dashboard/top-sql.md [hunk 3]
```diff
@@ -71,31 +68,39 @@ The following are the common steps to use Top SQL.
...
 
-* You can adjust the time range in the time picker or select a time range in the chart to get a more precise and detailed look at the problem. A smaller time range can provide more detailed data, with precision of up to 1 second.
+    Select a TiKV instance, and then select **By TABLE** or **By DB**:
```
[the remark this answers is not in the document]
Comment (lilin90) [reply dashboard/top-sql.md:97]: Agree.

**Review by nolouch — APPROVED** [review]

**Review by songrijie — APPROVED** [review]

**Review by lilin90 — COMMENTED** [review]
/approve

**Pull request merged**